### ISSUE TITLE
Non-numeric strings `toDouble` method resulting in `0.000000`

### ISSUE THREAD

**Issue by NathanC** [scala-native/scala-native#1103]
I've just started tinkering around with Scala Native (great project!) and tried compiling an existing sbt project of mine with it-- the only issue that I had is that the `toDouble` method behavior is not consistent with standard Scala. Strings like "x" or "foo" are resulting in a Double value of `0.000000` instead of `NumberFormatException`.

I modified `Hello.scala` in the project generated with `sbt new scala-native/scala-native.g8`, to demonstrate. Here is the modified program:

```
object Hello extends App {

  println(Try("100".toInt).toString)
  println(Try("foo".toInt).toString)
  
  println(Try("100.0".toDouble).toString)
  println(Try("foo".toDouble).toString)
}
```

And here is the output of running it after `nativeLink`:
```
Success(100)
Failure(java.lang.NumberFormatException: foo)
Success(100.000000)
Success(0.000000)
```

I'm using an updated 64 bit Arch Linux system. I'm not super familiar with native programming, so if there's some other critical information that would be helpful, just let me know. I might be able to take a stab at fixing this if someone can point me in the right direction.

**Comment by longshorej** [comment]
Hi @NathanC - this was fixed a couple months ago with #1047. It should be part of the latest release (out yesterday): https://github.com/scala-native/scala-native/releases

**Comment by densh** [comment]
Fixed in #1047.
